lib/zendesk_apps_support/assets/templates.js.erb

```diff
@@ -0,0 +1,8 @@
+<% unless templates.nil? %>
+{
+  <% templates.each do |name, fn| %>
+  <%= name %>: <%= fn %>,
```

Comment:
Hey @ocke, yep

I only began testing when I first put it up, realised there was a call missing, then moved onto something else :smile_cat: 

Thanks for the comment, much appreciated